docs/install/knative-with-operators.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Installing Knative with Knative Operators"
+weight: 10
+type: "docs"
+---
+
+# Knative installations with Knative Operators
+
+As of v0.10.0, Knative started to release Knative operators as tools to install, configure and manage Knative. This
+guide will explain how to install and uninstall Knative with Knative operators.
+
+There are two major components in Knative: Serving and Eventing. There are two operators: [Knative Serving Operator](https://github.com/knative/serving-operator)
+and [Knative Eventing Operator](https://github.com/knative/eventing-operator), respectively dedicated to each of them. Please make sure you have set up a Kubernetes
+cluster accessible to your local workstation.
+
+## Limitations of Knative Operators:
+
+Knative operators are still in Alpha phase. They only focus on Knative installation for the generic Kubernetes platform,
+which means Docker-Desktop Kubernetes on Mac and Minikube can be directly used as the Kubernetes clusters for you to install
+Knative with Knative operators, without any additional configuration out of the scope of the operator custom resource,
+but vendor-specific Kubernetes services, like Google Kubernetes Engine, IBM Kubernetes Service, OpenShift, etc, may need
+some additional configurations to work after the Knative is installed by the operator. In short, the Knative operators of
+the open source version only supports the following Kubernetes services, without additional configurations:
+
+   * Docker-Desktop Kubernetes on Mac
+   * Minikube
+
+The same to any other Kubernetes operator, Knative operators use custom resources as the sources of truth to configure
+your Knative. The current custom resources still have limited capabilities, which can not cover all the scenarios of
+how we can configure Knative to meet our needs, but Knative operators are making progress in extending their capabilities.
+One of the major missing points is that operators still lack of APIs for us to specify the parameters for high availability.
+If you rely on the HA capability, Knative operators are currently not helping with that.
```

Comment:
Change to something like
"Knative operators do no currently provide HA capabilities."